test/Staking.test.js

```diff
@@ -0,0 +1,299 @@
+      });
+      it("SetRewards false - Error : start time must be greater than current time", async () => {
+        let balanceBeforeOwner = await instanceToken.balanceOf(owner);
+        expect(balanceBeforeOwner).to.be.bignumber.equal(ether('5600000'));
+        let balanceBeforeContract = await instanceToken.balanceOf(instanceStaking.address);
+        expect(balanceBeforeContract).to.be.bignumber.equal(ether('0'));
+        let start = await time.latest();
+        await instanceToken.approve(instanceStaking.address, ether('500000'));
+        await expectRevert(instanceStaking.setRewards(start.add(new BN(0)), ether('500000'), ether('10')), "Error : start time must be greater than current time");
+        let balanceAfterOwner = await instanceToken.balanceOf(owner);
+        expect(balanceAfterOwner).to.be.bignumber.equal(ether('5600000'));
+        let balanceAfterContract = await instanceToken.balanceOf(instanceStaking.address);
+        expect(balanceAfterContract).to.be.bignumber.equal(ether('0'));
+      });
+      it("SetRewards false - Error : one of params equal to 0", async () => {
+        let balanceBeforeOwner = await instanceToken.balanceOf(owner);
+        expect(balanceBeforeOwner).to.be.bignumber.equal(ether('5600000'));
+        let balanceBeforeContract = await instanceToken.balanceOf(instanceStaking.address);
+        expect(balanceBeforeContract).to.be.bignumber.equal(ether('0'));
+        let start = await time.latest();
+        await instanceToken.approve(instanceStaking.address, ether('500000'));
+        await expectRevert(instanceStaking.setRewards(start.add(new BN(60)), ether('500000'), new BN(0)), "Error : one of params equal to 0");
+        let balanceAfterOwner = await instanceToken.balanceOf(owner);
+        expect(balanceAfterOwner).to.be.bignumber.equal(ether('5600000'));
+        let balanceAfterContract = await instanceToken.balanceOf(instanceStaking.address);
+        expect(balanceAfterContract).to.be.bignumber.equal(ether('0'));
+      });
+      it("SetRewards false - Ownable: caller is not the owner", async () => {
+        let start = await time.latest();
+        await expectRevert(instanceStaking.setRewards(start.add(new BN(60)), ether('500000'), new BN(0), { from: acc2 }), "Ownable: caller is not the owner");
+      });
+      it("Stake by acc2 false - Error : staking has not started yet", async () => {
+        let balanceBefore = await instanceToken.balanceOf(acc2);
+        expect(balanceBefore).to.be.bignumber.equal(ether('100'));
+        await instanceToken.approve(instanceStaking.address, ether('100'), { from: acc2 });
+        await expectRevert(instanceStaking.stake(ether('60'), { from: acc2 }), "Error : staking has not started yet");
+        let balanceAfter = await instanceToken.balanceOf(acc2);
+        expect(balanceAfter).to.be.bignumber.equal(ether('100'));
+      });
+    });
+
+    describe("Done", async () => {
```

Comment:
Rename Done to "Should success ...."

Rename Fail/Error to "Should fail if ...."